doc-downloader.ps1

```diff
@@ -29,11 +29,13 @@ If($DATED_FOLDERS) {
 Else {
     $FILE_DIR = $DESTINATION_PATH}
 
-$DRIVE_AXLE_HEADERS = @{ Authorization = ("driveaxle=" + $API_KEY) 
-                         Accept = 'application/json'}
+$DRIVE_AXLE_HEADERS = @{ Authorization = ("driveaxle=" + $API_KEY)
+                         Accept = 'application/json'
+                         ContentType = 'application/json'}
 
-$ELEOS_HEADERS = @{ Authorization = ("key=" + $API_KEY)
-                    Accept = 'application/json'}
+$ELEOS_HEADERS = @{ Authorization = ("Key key=" + $API_KEY)
```

Comment:
This seems like it should have a broken test associated with it.

Reply:
I thought so too. I don't think the mock server check authentication